File: bootstrap/src/integrationTest/resources/suite/exec/0002.hs
local log = require('log')
local sys = require('sys')
local test = require('test')

-- Remember: Each test is an adhoc invoked exec
local execs = sys.exec.list()
test.assert(execs.length == 1)

local test_exec = execs[1]

local exec = sys.exec.get(test_exec.id)
test.assert(exec.id == test_exec.id)
test.assert(exec.status == 'Started')
test.assert(exec.correlationId == nil)
test.assert(exec.inputs == {})

test.complete()

File: faas/bootstrap/src/integrationTest/resources/suite/exec/0002.lua
local sys = require('sys')
local test = require('test')
--
---- Remember: Each test is an adhoc invoked exec
local err, execs = sys.exec.list()
assert(err == nil)
assert(#execs == 1)

local test_exec = execs[1]

local err, exec = sys.exec.get(test_exec.id)
assert(err == nil)

assert(exec.id == test_exec.id)
assert(exec.status == 'Started')
assert(exec.correlationId == nil)
assert(#exec.inputs == 0)

test.complete()